docs/contribute/saros-testing-framework.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Saros Testing Framework (STF)
+---
+
+# {{ page.title }}
+{:.no_toc}
+
+{% include toc.html %}
+
+For further information
+about STF take a look at the [STF User
+Manual](http://saros-build.imp.fu-berlin.de/stf/STF_Manual.pdf).
```

Comment:
I do not know when the certificate expires and if it would be refresehd.

Reply:
The certificate is for our main build server hosted by the university. Even though we don't use it to build anymore, it still runs the XMPP server, meaning a valid certificate is important to securely access the web interface. It should get refreshed automatically (I think), otherwise we can just request a refresh when needed. But it was just recently refreshed (on October 15th) and is now valid until January 2021, so I think we have some headroom to use the HTTPS link with confidence ;)